lang/python/python-package.mk

```diff
@@ -130,7 +130,8 @@ define PyBuild/Compile/Default
 	$(foreach pkg,$(HOST_PYTHON_PACKAGE_BUILD_DEPENDS),
 		$(call host_python_pip_install_host,$(pkg))
 	)
-	$(call Build/Compile/PyMod,, \
+	$(call Build/Compile/PyMod,$(PYTHON_PKG_SETUP_SUBDIR), \
+		$(PYTHON_PKG_SETUP_PRE_ARGS) \
```

Comment:
could you give an example of using `PYTHON_PKG_SETUP_ARGS` ?
i don't have a clear mental picture of use-cases for this;

i'm not fully against adding it, but i'm not yet convinced about adding it yet

Reply:
PRE_ARGS is necessary in the case of python-lxc because I need to explicity run 
```
./setup.py build_ext --no-pkg-config install ...
```
rather than have build_ext just automagically called by install.

Reply:
oh build_ext also needs some -I and -L.

Reply:
in this case [with `build_ext`] it looks like maybe `build_ext` should be here [in `Build/Compile/Default` ] instead of `install`
and `install` moved into another build step [ maybe wrapped into `Package/$(1)/install` somehow ]

i remember [a while back] looking at this and wondering why the `./setup.py install` stuff is in `Build/Compile` ; maybe it's some left-over from the initial writers of this code; 
but, maybe it makes sense to do the split of `build_ext` + `install` now

Reply:
I'll try that and get back to you.

Reply:
I've split build_ext and install now -- still need to test though